imagecraft/application.py

```diff
@@ -33,6 +38,16 @@
 class Imagecraft(Application):
     """Imagecraft application definition."""
 
+    def __init__(
+        self,
+        app: craft_application.AppMetadata,
+        services: service_factory.ServiceFactory,
+    ) -> None:
+        super().__init__(app=app, services=services, extra_loggers={"imagecraft"})
+        self._global_args: dict[str, Any] = {}
```

Comment:
Other than the type, what's the difference between this and `self._global_arguments`? Are these the args passed by the dispatcher? If so, when will the Application need to access them?

Reply:
Good catch. See my other comments above. This is now removed.